Kitodo-MODSXML-SchemaConverter/src/main/java/org/kitodo/modsxmlschemaconverter/ModsXMLSchemaConverter.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.modsxmlschemaconverter;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.nio.charset.Charset;
+import java.util.Objects;
+import java.util.UnknownFormatConversionException;
+
+import javax.xml.XMLConstants;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.Result;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.sax.SAXResult;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.transform.sax.SAXTransformerFactory;
+import javax.xml.transform.sax.TransformerHandler;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kitodo.api.schemaconverter.DataRecord;
+import org.kitodo.api.schemaconverter.FileFormat;
+import org.kitodo.api.schemaconverter.MetadataFormat;
+import org.kitodo.api.schemaconverter.SchemaConverterInterface;
+import org.kitodo.exceptions.ConfigException;
+import org.xml.sax.InputSource;
+
+public class ModsXMLSchemaConverter implements SchemaConverterInterface {
+
+    private static final Logger logger = LogManager.getLogger(ModsXMLSchemaConverter.class);
+
+    private static final String XSLT_FILEPATH = "/xslt/mods2kitodo.xsl";
```

Comment:
Is this hard coded value intent? Later this value is used to retrieve a file from class path.